libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -199,6 +199,11 @@ enum PaymentTypeFilter {
     "ClosedChannel",
 };
 
+dictionary PaymentMetadata {
```

Comment:
NIT: rename to PaymentMetadataFilter

Reply:
Given the other reply about passing a vector to `set_payment_metadata`, maybe we should keep this as PaymentMetadata and use it for both filtering and inserting. What do you think?

Reply:
The filter structure might change in the future ( we may for example add support for operators (contains, equals, etc...).
I think it is better to have a separate structure even though currently they look the same, they represent different things.